### PR TITLE
Team Capes: Added minimum cape count

### PR DESCRIPTION
**Pull request by Matsyir**
Added minimum cape count to team cape plugin to reduce clutter/spam. Default is 1, so current behaviour is unchanged. First commit!

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/teamcapes/TeamCapesConfig.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.teamcapes;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup(
+		keyName = "teamCapes",
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/teamcapes/TeamCapesConfig.java]: Indentation is 1 tab

File: runelite-client/src/main/java/net/runelite/client/plugins/teamcapes/TeamCapesPlugin.java [hunk 2]
```diff
@@ -49,10 +48,25 @@
...
+	@Provides
+	TeamCapesConfig provideConfig(ConfigManager configManager)
+	{
+		return configManager.getConfig(TeamCapesConfig.class);
+	}
+
+	@Override
+	protected void startUp() throws Exception
+	{
+		teamCapesOverlay = new TeamCapesOverlay(this, config);
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/teamcapes/TeamCapesPlugin.java]: This should be injected as before

File: runelite-client/src/main/java/net/runelite/client/plugins/teamcapes/TeamCapesPlugin.java [hunk 3]
```diff
@@ -25,16 +25,15 @@
 package net.runelite.client.plugins.teamcapes;
 
 import java.time.temporal.ChronoUnit;
-import java.util.Comparator;
-import java.util.HashMap;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/teamcapes/TeamCapesPlugin.java]: No wildcards imports

File: runelite-client/src/main/java/net/runelite/client/plugins/teamcapes/TeamCapesPlugin.java [hunk 4]
```diff
@@ -25,16 +25,15 @@
...
 import java.util.stream.Collectors;
 import javax.inject.Inject;
+
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/teamcapes/TeamCapesPlugin.java]: Or groupings

**Comment by Matsyir** [comment]
Will make the necessary changes

**Pull request merged**